waterlog/src/App.scss

```diff
@@ -13,17 +13,24 @@
 #cost-buttons {
 	padding-top: 18px; 
 }
-
 .btn-graph-nav{
 	height:26px; 
 	width: 84px;
 }
- 
+#divide{
```

Comment:
empty lines between declarations (Do for entire file)

Reply:
Left lines for entire file